docs/apache-airflow-providers-amazon/secrets-backends/aws-ssm-parameter-store.rst

```diff
@@ -67,9 +67,10 @@ you would want to store your connection at ``/airflow/connections/smtp_default``
 Optionally you can supply a profile name to reference aws profile, e.g. defined in ``~/.aws/config``.
 
 The value of the SSM parameter must be the :ref:`connection URI representation <generating_connection_uri>`
-of the connection object.
+or in the :ref:`JSON Format <connection-serialization-json-example>` of the connection object.
 
-In some cases, URI's you will need stored in Secrets Manager may not be intuitive, for example when using HTTP / HTTPS or SPARK, you may need URI's that will look like this:
+In some cases, URI's you will need stored in AWS SSM Parameter Store may not be intuitive,
+for example when using HTTP / HTTPS or SPARK, you may need URI's that will look like this:
```

Comment:
Regarding the above thread, I think it's the use of past tense that makes it read oddly? What about this minor change: 
```suggestion
In some cases, URI's that you will need to store in AWS SSM Parameter Store may not be intuitive,
for example when using HTTP / HTTPS or SPARK, you may need URI's that will look like this:
```